docs/pages/application-access/okta/reference.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Okta Service Reference Documentation
+description: Configuration and CLI reference documentation for Teleport Okta service.
+---
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Okta Service
+appearing in the `teleport.yaml` configuration file:
+
+```yaml
+(!docs/pages/includes/config-reference/okta-service.yaml!)
+```
+
+## Okta import rule resources
+
+Full YAML spec of Okta import rule resources managed by `tctl` resource commands:
+
+```yaml
+kind: okta_import_rule
+version: v1
+metadata:
+  # Okta import rule name.
+  name: test-rule
+  # Okta import rule description.
```

Comment:
nit: I don't personally see the comments above name and description as necessary. Those are pretty obvious and it might be less noise if you just removed them.